--- configure.orig	2012-10-30 16:37:57.898112893 +0100
+++ configure	2012-10-30 16:38:01.591206104 +0100
@@ -6174,11 +6174,11 @@
 		# debug builds.
 		OPT="-g -O0 -Wall $STRICT_PROTO"
 	    else
-		OPT="-g $WRAP -O3 -Wall $STRICT_PROTO"
+		OPT="-g $WRAP -O0 -Wall $STRICT_PROTO"
 	    fi
 	    ;;
 	*)
-	    OPT="-O3 -Wall $STRICT_PROTO"
+	    OPT="-O0 -Wall $STRICT_PROTO"
 	    ;;
 	esac
 	case $ac_sys_system in
